com.inochi2d.inochi-creator.yml

```diff
@@ -0,0 +1,81 @@
+id: com.inochi2d.inochi-creator
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+# sdk-extensions:
+#   - org.freedesktop.Sdk.Extension.ldc
+#   - org.freedesktop.Sdk.Extension.dmd #This lacks aarch64 support, so gonna just pull a binary and later use only-arches to grab one when/if it's released.
+command: "inochi-creator"
+finish-args:
+    - "--device=all" #OpenGL rendering, webcams (may be required? not sure)
+    - "--socket=x11"
+    - "--share=ipc" # The X and Wayland sockets are permanently exposed, because kdialog takes longer to start if it only has X access, and doesn't work at all if only using Wayland.
+    - "--socket=wayland"
+    - "--filesystem=host" #This is a dev tool, it should probably have as much access to the host as it can. It also doesn't yet utilize portals.
+    - "--env=XDG_CURRENT_DESKTOP=KDE" #Hopefully fixes the issues it has with running on non-KDE desktops.
+
+#This extension will be needed in some newer releases, as of https://github.com/Inochi2D/inochi-creator/commit/5d67cded1273aced9033d327a8eb29c6f078a3c0
+# add-extensions:
+#   org.freedesktop.Platform.ffmpeg-full:
+#     version: '21.08'
+#     autodownload: true
+#     autodelete: false
+
+
+modules:
+  # --- Tools ---
+  - name: xprop
+    sources: 
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.4.tar.bz2
+        sha256: "8c77fb096e46c60032b7e2bde9492c3ffcc18734f50b395085a5f10bfd3cf753"
+
+  - name: zenity
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/zenity/3.41/zenity-3.41.0.tar.xz
+        sha256: 19b676c3510e22badfcc3204062d432ba537402f5e0ae26128c0d90c954037e1
+
+  - name: kdialog
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/KDE/kdialog.git
+        tag: v22.08.1
+
+
+  # --- Inochi Creator ---
+  - name: Inochi Creator
+    buildsystem: simple
+    build-commands:
+
+      - mv inochi-creator /app/bin/inochi-creator
+
+    sources: 
+      - type: archive
+        url: https://github.com/Inochi2D/inochi-creator/releases/download/v0.7.3/inochi-creator-linux-x86_64.zip
+        sha256: 026a2d8a4cf563ca5b9654da0dc21a3f715eaeeacc865f34d0d64511e59435e8 
+
+  - name: Metadata
+    buildsystem: simple
+    build-commands: 
+      - patch -u ./res/inochi-creator.appdata.xml -i metadata.patch
+      - patch -u ./res/inochi-creator.desktop -i desktopfile.patch
+

```

Comment:
this not how you apply patches.

Reply:
How would i go about patching them? I just generated a quick patchfile, and this worked for me locally. I'm not quite sure on how the `patch` source type works.

Reply:
I would really appreciate some help with this. I supplied some modified patch files similar to how the Godot baseapp does it, but it can't find the files to patch.

https://github.com/flathub/org.godotengine.godot.BaseApp/blob/branch/3.5/godot-i386-disable-O3.patch